docs/tutorial/visualization.rst

```diff
@@ -0,0 +1,420 @@
+    vertices = mesh.vertex_sample(size=4)
+    edges = mesh.edge_sample(size=4)
+    faces = mesh.face_sample(size=2)
+
+    Artist(mesh).draw(vertices=vertices, edges=edges, faces=faces)
+
+
+Colors can be modified globally per element type,
+
+.. code-block:: python
+
+    from compas.artists import Artist
+    from compas.geometry import Frame, Box
+    from compas.datastructures import Mesh
+
+    vertices = mesh.vertex_sample(size=4)
+    edges = mesh.edge_sample(size=4)
+    faces = mesh.face_sample(size=2)
+
+    Artist(mesh).draw(vertices=vertices, edges=edges, faces=faces, vertexcolor=(255, 0, 0), edgecolor=(0, 255, 0), facecolor=(0, 0, 255))
+
+
+or individually per element.
+
+.. code-block:: python
+
+    import random
+    from compas.artists import Artist
+    from compas.geometry import Frame, Box
+    from compas.datastructures import Mesh
+    from compas.utilities import i_to_rgb
+
+    vertices = mesh.vertex_sample(size=4)
+    edges = mesh.edge_sample(size=4)
+    faces = mesh.face_sample(size=2)
+
+    vertex_color = {vertex: i_to_rgb(random.random()) for vertex in vertices}
+    edge_color = {edge: i_to_rgb(random.random()) for edge in edges}
+    face_color = {face: i_to_rgb(random.random()) for face in faces}
+
+    Artist(mesh).draw(vertices=vertices, edges=edges, faces=faces, vertexcolor=vertex_color, edgecolor=edge_color, facecolor=face_color)
+
+
+Redraw
+======
+
+In Rhino, automatic redrawing of the view has been turned off
+such that the view is not continuously updated with every object that is added to the scene.
+
+In most cases, a redraw of the view is called automatically called at the end of the execution of a script.
+The only exception to that is when a script is executed using the built-in script editor on Mac.
+In that context, an explicit call to ``Artist.redraw()`` is needed to prevent the view from freezing up.
+
+.. code-block:: python
+
+    from compas.geometry import Frame, Box
+    from compas.artists import Artist
+
+    box = Box(Frame.worldXY(), 1, 1, 1)
+
+    Artist(box).draw()
+
+    Artist.redraw_scene()
+
+
+The ``redraw`` method can also be used for interactive scripts or dynamic visualizations,
+where the view has to be redrawn at potentially multiple stages before the end of the script execution.
+
+.. code-block:: python
+
+    import time
+    from compas.geometry import Frame, Box, Translation
+    from compas.artists import Artist
+
+    box = Box(Frame.worldXY(), 1, 1, 1)
+
+    artist = Artist(box)
+    artist.draw(redraw_scene=True)
+
+    T = Translation.from_vector([1, 0, 0])
+
+    for i in range(10):
+        time.sleep(1)
+        box.transform(T)
+        artist.draw(redraw_scene=True)
+
+
+Clear
+=====
+
+Sometimes the entire scene has to be cleaned before drawing any new objects.
+This can be the case, for example, when running a script multiple times in a row to test different versions of a WIP algorithm or procedure.
+
+.. code-block:: python
+
+    Artist.clear_scene()
```

Comment:
`scene` is a very overloaded term, especially with the things we have in the pipeline.
So, I think we need another name for this, `clear_all`?

Reply:
`clear_view`?

Reply:
sounds good 👍